app/views/admin/topics/_ticket.html.erb

```diff
@@ -6,7 +6,7 @@
 
   <div id="ticket-controls" class="col-md-8 col-xs-12 col-lg-8">
     <h2>
-      <%= render 'admin/topics/ticket_nav_dropdown' %> #<%= @topic.id %>- <%= @topic.name -%>
+      <%= render 'admin/topics/ticket_nav_dropdown' %> #<%= @topic.id %>- <%= best_in_place @topic, :name, as: :input, activator: :span -%>&nbsp;<span class="btn" id="btn_edit_topic_name">Edit</span>
```

Comment:
Good catch @CGA1123 I didn't notice this when I looked at it the first time.  This should be exposed to admins only.